docs/DeveloperGuide.md

```diff
@@ -343,29 +349,32 @@ In GuestBook, the user can mark all the rooms as unclean. This feature was added
 <span style="font-size: large; color: #e46c0a">Implementation:</span>
 * The `markroomsunclean` command edits all the guests in GuestBook and changes their `isRoomClean` statuses to `no`. It takes in no additional inputs or fields.
 
-The following activity diagram summarises what happens when a user enters a `markroomsunclean` command.
+The following activity diagram summarizes what happens when a user enters a `markroomsunclean` command.
 
 ![MarkRoomsUncleanActivityDiagram](images/MarkRoomsUncleanActivityDiagram.png)
-* When the user executes the `markroomsunclean` command, GuestBook will retrieve the list of all the guests that requires editing of the `isRoomClean` status.
+* When the user executes the `markroomsunclean` command, GuestBook will retrieve the list of all the guests in GuestBook.
 * A new list of guests will be created with `isRoomClean` fields set to `no`. The other details of the guests will remain unchanged.
 * Finally, the new list of guests is set to the model.
 
 <span style="font-size: large; color: #e46c0a">Design Considerations:</span>
+
+
 **Aspect: The scope at which the command changes all guests' `isRoomClean` statuses**
-* Alternative 1: Allow `markroomsunclean` command to operate only on the last shown list instead of the entire list. This is to standardise how edits are made across the commands (e.g. `edit` and `delete`).
+* **Alternative 1:** Allow `markroomsunclean` command to operate only on the last shown list instead of the entire guest list. This is to standardise how edits are made across the commands (e.g. `edit` and `delete`).
   * Pros: This might be more intuitive for users, as `edit` and `delete` commands work only on the last shown lists.
   * Cons: User is unable to change all the guests' `isRoomClean` statuses in a single command.
-* Alternative 2 (current choice): Allow `markroomsunclean` command to change all guests' `isRoomClean` statuses in GuestBook instead of the last shown list.
-  * Pros: User is able to change all the guests' `isRoomClean` statuses in a single command.
+* **Alternative 2 (current choice):** Allow `markroomsunclean` command to change all guests' `isRoomClean` statuses in GuestBook instead of the last shown list.
+  * Pros: User is able to change all the guests' `isRoomClean` statuses in a single command. This improves efficiently for users.
   * Cons: There is less flexibility in marking groups of guests' rooms as unclean.
 
 Taking into consideration the context of GuestBook that operates for small hotels, it is unlikely to have a case in which the user has to mark different groups of guests' `isRoomClean` statuses differently as the types of rooms as mostly homogenous. Hence, we decided to proceed with Alternative 2.
```

Comment:
For our CS2101 review, a group mentioned that "homogenous" might be difficult to understand what exactly it means. Do you think it is alright or should we phrase it differently?

Reply:
Yes, will fix!